api/client/network.go

```diff
@@ -46,6 +47,8 @@ func (cli *DockerCli) CmdNetworkCreate(args ...string) error {
 	cmd.Var(flIpamAux, []string{"-aux-address"}, "auxiliary ipv4 or ipv6 addresses used by Network driver")
 	cmd.Var(flOpts, []string{"o", "-opt"}, "set driver specific options")
 
+	flInternal := cmd.Bool([]string{"-internal"}, false, "configs the network to be internal which does not have access to the external world")
```

Comment:
a minor nit. The description seems verbose. pls consider changing it to `restricts external access for the network`